src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -57,12 +57,12 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
 
     /// @notice the current number of quorums supported by the registry coordinator
     uint8 public quorumCount;
-    /// @notice the mapping from quorum number to a quorums operator cap and kick parameters
-    mapping(uint8 => OperatorSetParam) internal _quorumOperatorSetParams;
-    /// @notice the mapping from operator's operatorId to the updates of the bitmap of quorums they are registered for
-    mapping(bytes32 => QuorumBitmapUpdate[]) internal _operatorIdToQuorumBitmapHistory;
-    /// @notice the mapping from operator's address to the operator struct
-    mapping(address => Operator) internal _operators;
+    /// @notice maps quorum number => operator cap and kick params
+    mapping(uint8 => OperatorSetParam) internal _quorumParams;
+    /// @notice maps operator id => historical quorums they registered for
+    mapping(bytes32 => QuorumBitmapUpdate[]) internal _operatorBitmapHistory;
+    /// @notice maps operator address => operator id and status
+    mapping(address => Operator) internal _operatorInfo;
```

Comment:
should we perhaps name the struct 'OperatorInfo' as well?

Reply:
+1. Will make this change in a future pr.